charts/stable/bytestash/Chart.yaml

```
@@ -33,6 +33,6 @@ name: bytestash
 sources:
   - https://ghcr.io/jordan-dalby/bytestash
   - https://github.com/jordan-dalby/ByteStash
-  - https://github.com/truecharts/charts/tree/master/charts/incubator/bytestash
+  - https://github.com/truecharts/charts/tree/master/charts/stable/bytestash
 type: application
-version: 0.0.1
+version: 0.0.2
```

```suggestion
version: 1.0.0
```
For stable move. We bump it to 1.0.0